packages/vue/src/vue-component-lib/utils.ts

```diff
@@ -1,7 +1,7 @@
 import { VNode, defineComponent, getCurrentInstance, h, inject, ref, Ref } from 'vue';
 
-export interface InputProps {
-  modelValue?: string | boolean;
+export interface InputProps<T> {
```

Comment:
Can we remove this? Looks like the vue output target changes got into this PR.

Reply:
Ahh yep, didn't re-install our lock version after experimenting with a dev-build. I'll revert this change.